plugins/network-elements/nuage-vsp/src/com/cloud/agent/api/guru/ReserveVmInterfaceVspCommand.java

```diff
@@ -19,25 +19,29 @@
 
 package com.cloud.agent.api.guru;
 
-import com.cloud.agent.api.Command;
+import net.nuage.vsp.acs.client.api.model.VspDhcpVMOption;
 import net.nuage.vsp.acs.client.api.model.VspNetwork;
 import net.nuage.vsp.acs.client.api.model.VspNic;
 import net.nuage.vsp.acs.client.api.model.VspStaticNat;
 import net.nuage.vsp.acs.client.api.model.VspVm;
 
+import com.cloud.agent.api.Command;
+
 public class ReserveVmInterfaceVspCommand extends Command {
 
     private final VspNetwork _network;
     private final VspVm _vm;
     private final VspNic _nic;
     private final VspStaticNat _staticNat;
+    private final  VspDhcpVMOption _dhcpOption;
```

Comment:
``private final VspNetwork`` and ``private final VspNetwork _network`` appear many of these command classes.  Would it make it sense to consolidate these commonalities in a VSP abstract command class?